README.md

```diff
@@ -1,22 +1,17 @@
-# Substrate Cumulus Parachain Template
+# Contracts Parachain Template
 
-A new [Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus)-based Substrate node, ready for hacking ☁️..
+A new [Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus)-based Substrate node, configured to supports Wasm-based contracts.
```

Comment:
```suggestion
A new [Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus)-based Substrate node, configured to support Wasm-based smart contracts.
```

I see the same typo in the description in pop-cli, so it needs to be fixed there as well.